### ISSUE TITLE
Bitcoin Core says it's the first time I am opening the program when in fact I've previously downloaded the blockchain until Dec 27, 2020. how do I resume syncing?

### ISSUE THREAD

**Issue by jm9151821** [bitcoin-core/gui#170]
Not sure if this is the right place to put this but basically, I've already downloaded the blockchain until Dec 27, but now when I open bitcoin core on my hp windows laptop.  I got the popup saying this is the first time I launch bitcoin core, which it isn't, but then I selected the 'chainstate' folder on my external hard drive, hoping it would resume the blockchain but instead it started downloading everything from scratch again. 

TLDR: Bitcoin Core is asking where to store the blockchain, despite me already having downloaded the blockchain until Dec 27,2020 and I am wondering how to resume the syncing from where I have downloaded. The program starts to redownload the whole blockchain.

**Comment by MarcoFalke** [comment]
Usually the issue tracker is used to track technical issues related to the Bitcoin Core code base. Keep in mind that general bitcoin questions and/or support requests are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com) or the `#bitcoin` IRC channel on freenode.

**Comment by hebasto** [comment]
Closing?

**Comment by jm9151821** [comment]
Yes, it was fixed with adding '--' to the directory and it picked back up
the directory. I was able to find the answer.
Thanks
